x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -276,5 +315,422 @@ export function getDetailsItems(anomaly: AnomaliesTableRecord, filter: EntityCel
+            defaultMessage="Anomaly explanation"
+          />
+        </h4>
+      </EuiText>
+      <EuiSpacer size="s" />
+
+      {explanationDetails.map(({ title, description }) => (
+        <>
+          <EuiFlexGroup gutterSize="none">
+            <EuiFlexItem style={{ width: '220px' }} grow={false}>
+              {title}
+            </EuiFlexItem>
+            <EuiFlexItem>{description}</EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="xs" />
+        </>
+      ))}
+
+      <EuiSpacer size="s" />
+      {impactDetails.length ? (
+        <>
+          <EuiText size="xs">
+            <h4>
+              <FormattedMessage
+                id="xpack.ml.anomaliesTable.anomalyDetails.impactOnScoreTitle"
+                defaultMessage="Impact on initial score"
+              />
+            </h4>
+          </EuiText>
+          <EuiSpacer size="s" />
+
+          {impactDetails.map(({ title, description }) => (
+            <>
+              <EuiFlexGroup gutterSize="none">
+                <EuiFlexItem css={{ width: '220px' }} grow={false}>
+                  {title}
+                </EuiFlexItem>
+                <EuiFlexItem>{description}</EuiFlexItem>
+              </EuiFlexGroup>
+              <EuiSpacer size="xs" />
+            </>
+          ))}
+        </>
+      ) : null}
+    </div>
+  );
+};
+
+const RecordScore: FC<{ score: number }> = ({ score }) => {
+  return (
+    <div
+      css={{
+        borderBottom: '2px solid',
+      }}
+      style={{
+        borderBottomColor: getSeverityColor(score),
+      }}
+    >
+      {score}
+    </div>
+  );
+};
+
+function getAnomalyType(explanation: MLAnomalyDoc['anomaly_score_explanation']) {
+  if (
+    explanation === undefined ||
+    explanation.anomaly_length === undefined ||
+    explanation.anomaly_type === undefined
+  ) {
+    return null;
+  }
+
+  const dip = i18n.translate('xpack.ml.anomaliesTable.anomalyExplanationDetails.anomalyType.dip', {
+    defaultMessage: 'Dip over {anomalyLength, plural, one {# bucket} other {# buckets}}',
+    values: { anomalyLength: explanation.anomaly_length },
+  });
+  const spike = i18n.translate(
+    'xpack.ml.anomaliesTable.anomalyExplanationDetails.anomalyType.spike',
+    {
+      defaultMessage: 'Spike over {anomalyLength, plural, one {# bucket} other {# buckets}}',
+      values: { anomalyLength: explanation.anomaly_length },
+    }
+  );
+
+  return explanation.anomaly_type === 'dip' ? dip : spike;
+}
+
+function getImpactValue(score: number) {
+  if (score < 1) return 1;
+  if (score < 3) return 2;
+  if (score < 6) return 3;
+  if (score < 12) return 4;
+  return 5;
+}
+
+const impactTooltips = {
+  anomaly_characteristics: {
+    low: i18n.translate(
+      'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyCharacteristicsTooltip.low',
+      {
+        defaultMessage:
+          'The statistical properties of the detected anomalous interval have a low impact on the score.',
```

Comment:
@valeriy42 are there any more details we can provide for the 'statistical properties' info? This text is a bit unclear to me and leaves me wondering 'what statistical properties'.

Reply:
"The shape of the anomaly distribution"

Reply:
magnitude, scale, extent, expanse, range of deviation

Reply:
How about this?

```
Moderate /  medium / high impact from the duration and magnitude of the detected anomaly relative to the historical average.
```